stanford/roles/deployment/defaults/main.yml

```diff
@@ -9,10 +9,11 @@ deployment_subnet_index: "{{ (CLUSTER_NUMBER - 1 |int|abs) % 2 |int }}"
 DEPLOYMENT_NUMBER: "{{ deployment_subnet_index|int + 1 | int }}"
 DEPLOYMENT_SUBNET_PUBLIC: "{{ DEPLOYMENT_SUBNETS['public'][deployment_subnet_index|int] }}"
 DEPLOYMENT_SUBNET_PUBLIC_ID: "{{ deployment_subnet_public.subnet.id }}"
```

Comment:
And this one goes away with https://github.com/Stanford-Online/configuration/pull/232